roles/common/tasks/host_validations.yml

```diff
@@ -78,14 +78,110 @@
   tags:
   - validate
   - validate_tmp_access
-  
-  - name: Validate enough free disk space for package installation
+
+- name: Validate enough free disk space for package installation
   assert:
     that: item.size_available > {{ required_disk_space_mb }}|float
-    fail_msg: >- 
-      "Not enough free disk space for package installation.  Minimum space required is {{ required_disk_space_mb }}MB, please free up more disk space 
+    fail_msg: >-
+      "Not enough free disk space for package installation.  Minimum space required is {{ required_disk_space_mb }}MB, please free up more disk space
       to continue installation. To skip host validations, set validate_hosts to false."
   with_items: "{{ ansible_mounts }}"
-  tags: 
+  tags:
     - validate
     - validate_disk_usage
+
+- name: Validate enough free memory for Zookeeper hosts
+  assert:
+    that: ansible_memtotal_mb > {{ required_total_memory_mb_zookeeper }}|float
```

Comment:
Does it not make sense to use * ansible_memfree_mb*

Reply:
Based on our internal discussions it does not.  As when discussed with product, this is to be treated as an installer to make sure that basic hardware requirements are met, and not about the utilisation of said hardware.

Reply:
Thinking about this further, I think you are right.  Let me update this.